package.json

```diff
@@ -36,6 +32,10 @@
     "contributors:add": "all-contributors add",
     "contributors:generate": "all-contributors generate"
   },
+  "resolutions": {
```

Comment:
This is temporary to make sure CI passes again for the `ember-beta` scenario (and these are all dev dependencies). `ember-freestyle` itself should now be completely deprecation free and runs against Ember v4.